react/ShelfContent.js

```diff
@@ -94,6 +96,19 @@ class ShelfContent extends Component {
     )
   }
 
+  pushPixelProductsImpressions = () => {
+    const { products } = this.props
+
+    products.forEach((product, index) => {
+      this.props.push({
+        event: 'productImpression',
+        list: 'Home Page Shelf',
```

Comment:
Since this is not a required field and the shelf is not always the home shelf, I think we could remove this for now.

Reply:
Or mantaining just _Shelf_. Could be strategically interesting.